src/odemis/acq/test/path_test.py

```diff
@@ -52,10 +52,49 @@
 SPARC2_CONFIG = CONFIG_PATH + "sim/sparc2-sim.odm.yaml"
 SPARC2_EXT_SPEC_CONFIG = CONFIG_PATH + "sim/sparc2-ext-spec-sim.odm.yaml"
 SECOM_FLIM_CONFIG = CONFIG_PATH + "sim/secom-flim-sim.odm.yaml"
-SPARC2_POLARIZATIONANALYZER_CONFIG = CONFIG_PATH + "sim/sparc2-polarizer-sim.odm.yaml"
+SPARC2_POLARIZATIONANALYZER_CONFIG = CONFIG_PATH + "sim/sparc2-ek-polarizer-sim.odm.yaml"
 SPARC2_4SPEC_CONFIG = CONFIG_PATH + "sim/sparc2-4spec-sim.odm.yaml"
 
 
+def path_pos_to_phys_pos(pos, comp, axis):
+    """
+    Convert a generic position definition into an actual position (as reported in comp.position)
+    pos (float, str, or tuple): a generic definition of the position
+    comp (Actuator)
+    axis (str): the axis for which the position is read
+    """
+    axis_def = comp.axes[axis]
+
+    md = comp.getMetadata()
+    if isinstance(pos, tuple):  # multiple options
+        for p in pos:
+            # From the metadata?
+            if isinstance(p, str) and p.startswith("MD:"):
+                md_name = p[3:]
+                try:
+                    pos = md[md_name][axis]
+                    logging.debug("Found metadata %s[%s]= %s", md_name, axis, pos)
+                    break
+                except KeyError:  # no such metadata, or axis on the metadata
```

Comment:
Is it not necessary to do debug logging when a KeyError exception occurs?

Reply:
Not really, this is actually very common.
However, the logging.debug() above should go away. That is left over from my debugging, but will fill up the testing log unnecessarily.